resource-types/GithubStatus.dhall

```diff
@@ -0,0 +1,52 @@
+{ meta = { name = "github-status", repository = "resource/github-status" }
+, Version = { Type = {}, default = {=} }
+, Params =
+  { Get =
+    { Type = { path : Text, context : Optional Text }
+    , default.context = None Text
+    }
+  , Put =
+      let StatusObject =
+            { Type =
+                { state : Text
+                , context : Optional Text
+                , description : Optional Text
+                , target_url : Optional Text
+                }
+            , default =
+              { context = None Text
+              , description = None Text
+              , target_url = None Text
+              }
+            }
+
+      let Statuses =
+            { Type = { path : Text, statuses : List StatusObject.Type }
+            , default = {=}
+            }
```

Comment:
`Statuses` should also export `StatusObject` so that a user can specify e.g.

```dhall
Put.Options.Statuses::{
  path = "foo",
  statuses =
  [ Put.Options.Statuses.StatusObject::{
    context = "bar"
    }
  ]
}
```